README.md

```diff
@@ -1,97 +1,40 @@
-# HTML & CSS3 Course
+# Hello-microverse project
 
-If you are not familiar with linters, read [root level README](../README.md).
+In this project, I set up a "Hello Microverse" repository. No complex coding was required for this exercise. My goal was to master all of the tools and best practices I learned about in previous steps.
 
-## Set-up GitHub Actions
```

Comment:
- [ ] Great job using [Microverse Template](https://github.com/microverseinc/readme-template) however please add table of content from the template into your README and also please make sure all the main sections from table of content are included in your README. The table of contents serves two purposes: It gives users an overview of the document's contents and organization. It allows readers to go directly to a specific section of an on-line document.